platform/ui/src/components/StudyListFilter/StudyListFilter.tsx

```diff
@@ -11,6 +11,7 @@ const StudyListFilter = ({
   clearFilters,
   isFiltering,
   numOfStudies,
+  onUploadClick,
```

Comment:
`onUploadClick` sounds to me like an eventHandler, an action, but seems like it is a flag to render ICON?

Reply:
nevermind